dev/InfoBar/InfoBar.xaml

```diff
@@ -136,15 +136,15 @@
                             <primitives:InfoBarPanel
                                 Grid.Column="1"
                                 Margin="{StaticResource InfoBarPanelMargin}"
-                                primitives:InfoBarPanel.HorizontalMargin="{StaticResource InfoBarPanelHorizontalLayoutMargin}"
-                                primitives:InfoBarPanel.VerticalMargin="{StaticResource InfoBarPanelVerticalLayoutMargin}">
+                                PaddingInHorizontalOrientation="{StaticResource InfoBarPanelPaddingInHorizontalOrientation}"
+                                PaddingInVerticalOrientation="{StaticResource InfoBarPanelPaddingInVerticalOrientation}">
 
                                 <TextBlock x:Name="Title"
                                     Grid.Column="0"
                                     Text="{TemplateBinding Title}"
                                     Foreground="{ThemeResource InfoBarTitleForeground}"
-                                    primitives:InfoBarPanel.HorizontalMargin="{StaticResource InfoBarTitleHorizontalLayoutMargin}"
-                                    primitives:InfoBarPanel.VerticalMargin="{StaticResource InfoBarTitleVerticalLayoutMargin}"
+                                    primitives:InfoBarPanel.SpacingInHorizontalOrientation="{StaticResource InfoBarTitleSpacingInHorizontalOrientation}"
```

Comment:
@teaP, this TextBlock is the child of an InfoBarPanel, does it need to be setting the Grid.Column attached property?

Reply:
This is probably left over from before there was an InfoBarPanel. I don't see InfoBarPanel using these properties, so it should be safe to remove.

Reply:
They don't make a difference, so I removed them now.